backend/app/api/endpoints/adapter/tasks.py

```
@@ -320,6 +321,301 @@ async def cancel_task(
+            affected_subtasks.append(
+                {
+                    "subtask_id": subtask.id,
+                    "previous_status": "RUNNING",
+                    "new_status": "FAILED",
+                }
+            )
+            logger.info(
+                f"[CleanupOrphaned] Marked subtask {subtask.id} as FAILED, "
+                f"preserved_result={bool(existing_result)}, has_cached_content={bool(cached_content)}"
+            )
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)